docs/deploy_streamlit_app.md

```diff
@@ -1,3 +1,156 @@
-# Deploy a Streamlit app
+# Deploy an app
```

Comment:
I think the filename here should follow the pattern we use for the "Create an app" file. So `deploy_an_app.html`

Or, even better, we could rename this to `deploy.md`, rename the other page to `create.html`.

This is not just nitpicking, it's also visible to users via the URL.

(The exact pattern we go with is up to you, but I think the two pages should match.)

LMK what you land on, so I can link this from within the S4A app. Please ping me with whatever you decide!

Reply:
I have been naming the URLs in such a way that they had keyword phrases people were likely to type:

- deploy streamlit app
- develop streamlit components
- publish streamlit components
- streamlit components
- streamlit configuration
- streamlit faq 

I'm fine with revisiting this later if we don't think this is necessary from an SEO perspetive, but I don't think this is launch critical.